### DISCUSSION TITLE
knowledge transfer to GPT model

### DISCUSSION THREAD

**Discussion by haowang-bioinfo** [SysBioChalmers/Human-GEM#454] [Ideas]
how to transform the knowledge of GEMs into GPT language models?

**Comment by mihai-sysbio** [comment]
What purpose do you have in mind (curation, prediction)?

**Comment by haowang-bioinfo** [comment]
An idea is to train model from Human-GEM in a way similar to obtain [Codex](https://openai.com/blog/openai-codex/), and apply it for knowledge extraction.
